gen/definitions/keychain_classic.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: keychain_classic
```

Comment:
Just `Keychain`? In general we try to align with the CLI naming terminology to make it slightly more intuitive for users new to DME.